syncwatcher.go

```diff
@@ -194,6 +197,11 @@ func init() {
 		os.Exit(0)
 	}
 
+	if accumulationInterval != 500 {
```

Comment:
Just for my education: The problem is with the second occurrence of the "hardcoded" number 500 which might be missed if the standard is changed e.g. above in the flags, correct?